model/pdata/metrics.go

```diff
@@ -321,3 +321,30 @@ func (ms NumberDataPoint) Value() float64 {
 func (ms NumberDataPoint) SetValue(v float64) {
 	ms.SetDoubleVal(v)
 }
+
+// Type returns the type of the value for this Exemplar.
+// Calling this function on zero-initialized Exemplar will cause a panic.
+func (ms Exemplar) Type() NumberDataPointType {
```

Comment:
NumberDataPointType name is not good when using with Exemplars, maybe "NumberType"? or "MetricValueType"?

Reply:
I think `MetricValueType` makes the most sense, since `NumberType` seems too closely related to `NumberDataPoint`